### PR TITLE
feat(pie-icons): DSW-1750 add new icons

### PR DESCRIPTION
**Pull request by raoufswe**
## Describe your changes (can list changeset entries if preferable)

[Added] - new icons

### Misc
- qr-code
- qr-code-large

### Functionality
- restricted-filled
- restricted-filled-large

### Pay
- coins 
- coins-large 
- coins-filled-large 
- coins-filled

### Currency
- conversion-rate-filled
- conversion-rate-filled-large

### Payment
- 10bis
- bank-of-ireland-uk
- bank-of-scotland
- barclays
- danske-bank
- first-direct
- halifax
- hsbc
- lloyds
- monzo
- nationwide
- natwest
- revolut
- royal-bank-of-scotland
- santander
- starling
- tesco-bank
- tsb
- ulster-bank

## Author Checklist (complete before requesting a review)
- [x] I have performed a self-review of my code

### CODE REVIEW

**Review by LTurns — DISMISSED** [review]

File: packages/tools/pie-icons/src/assets/coins-filled.svg [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<svg width="17" height="16" viewBox="0 0 17 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (dandel10n) [inline packages/tools/pie-icons/src/assets/coins-filled.svg:1]: If it is a PIE icon it should have size 32x32 for large and 16x16 for small icons

File: packages/tools/pie-icons/src/assets/qr-code.svg [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<svg width="17" height="16" viewBox="0 0 17 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (dandel10n) [inline packages/tools/pie-icons/src/assets/qr-code.svg:1]: If it is a PIE icon it should have size 32x32 for large and 16x16 for small icons

File: packages/tools/pie-icons/src/assets/qr-code-large.svg [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<svg width="33" height="32" viewBox="0 0 33 32" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (dandel10n) [inline packages/tools/pie-icons/src/assets/qr-code-large.svg:1]: If it is a PIE icon it should have size 32x32 for large and 16x16 for small icons

File: packages/tools/pie-icons/src/assets/coins-filled-large.svg [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<svg width="33" height="32" viewBox="0 0 33 32" fill="none" xmlns="http://www.w3.org/2000/svg">
```
[the remark this answers is not in the document]
Comment (raoufswe) [reply packages/tools/pie-icons/src/assets/coins-filled-large.svg:1]: @tingting-lau @mollynye can you please check if this is expected or an oversight?

**Review by dandel10n — COMMENTED** [review]
Looks good, but few icons don't follow PIE icons size requirements

**Review by dandel10n — APPROVED** [review]

**Review by kevinrodrigues — APPROVED** [review]

**Pull request merged**